src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -893,7 +893,9 @@ javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* description?
 
       | VERSION_LITERAL (WS | NEWLINE)* description?
 
-      | CUSTOM_NAME (WS | NEWLINE)* description?
+      | CUSTOM_NAME (WS | NEWLINE)*
+        ({!isNextJavadocTag()}? LEADING_ASTERISK | htmlComment
+        | CDATA | NEWLINE | text | javadocInlineTag | htmlElement)*
```

Comment:
@Vladlis , if custom bock tags are affected the same way as inline tag, we need to fix them too, but in separate issue - please create if required.